gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Ui avatars squad

### Description

Made changes to the UI for squads and avatars.

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
python manage.py runserver
```

1. Goto the squads page and add people to your squad using e-mail.
2. Goto avatars page by clicking on the avatar on Profile page. Also check delete avatars page

### Miscellaneous Comments

@jawooson  I'm really sorry I couldn't do all the stuff I wanted to on your page. I wanted to work with displaying multiple avatars, they are stuck together and that's annoying. I spent a lot of time trying to work with it, but I think I might need your help to understand how django-avatars work. 

@mksavic  I made changes to the squads UI. Please take a look at it and let me know if that is good enough.

@vmoros @mksavic @jawooson @kavinshah  I would like to change the profile page. Remember Dan wrote a comment saying that he doesn't understand why there is a profile page and a Home page. So I thought of taking off the collapse on Profile page and that would only display the events and its information, we dont have to click on the event to view its details. I think that is kinda misleading having same collapses in different pages. Lmk if you guys agree with this.
### Checklist
